ansible_collections/arista/cvp/molecule/cv_image_v3/test_cv_image_v3.yaml

```diff
@@ -0,0 +1,164 @@
+---
+- name: Test cv_image_v3
+  hosts: CloudVision
+  connection: local
+  gather_facts: no
+  vars:
+    # Make sure given image does not exist on CVP
+    image_name: "EOS-4.25.4M.swi"
+  tasks:
+
+  ### Run all the tasks sequentially ###
+
+    - name: "Create bundle {{inventory_hostname}}"
+      vars:
+        ansible_command_timeout: 1200
```

Comment:
for these vars I think we can just move it to the top to make them global and then we won't need to specify it in each task

Reply:
Fixed!